docs/conditional.rst

```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
         views = NumberAttribute(default=0)
 
 
-AND vs. OR
-^^^^^^^^^^
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
 
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+PynamoDB supports creating condition expressions from attributes using a simple syntax. Attributes can be compared against values.
+Any value provided will be serialized using the serializer defined for that attribute.
 
 .. code-block:: python
 
-    thread_item = Thread('Existing Forum', 'Example Subject')
+    print(Thread.forum_name == 'Some Forum')
```

Comment:
Since we are supporting 2.7 with these changes, we should make the documentation python 2.7/3 agnostic. So it would be better to use `str`.

Also, I would find it useful to document the result of commands rather than me pasting them into the interpreter.